tests/run-build.sh

```diff
@@ -2,69 +2,68 @@
 
 set -ex
 
-# Skip build if requested
-if test -e ci-skip ; then
+function cleanup {
+    if [[ "${POSTCLEANUP}" == true ]]; then
+        for mp in $(mount | grep rkt | awk '{print $3}' | tac); do
+            sudo umount "${mp}"
+        done
+
+        for link in $(ip link | grep rkt | cut -d':' -f2); do
+            sudo ip link del "${link}"
+        done
+        sudo rm -rf /var/lib/cni/networks/*
```

Comment:
run-build.sh can be executed by people outside of CI throw-away VMs. It should not do destructive changes. It can be guarded by `if [ "$CI" = true ] ; then`.

[Semaphore](https://semaphoreci.com/docs/available-environment-variables.html) and others usually define $CI=true.